gamemodes/terrortown/gamemode/shared/lang/japanese.lua

```diff
@@ -0,0 +1,1274 @@
+---- Japanese language strings
+
+local L = LANG.CreateLanguage("japanese")

```

Comment:
all the other languages defined their language with their name. So for example "deutsch" for "german". Maybe you want to replace this as well?